mise.toml

```diff
@@ -43,28 +43,24 @@ EOF
 [tasks.deps]
 description = "install npm dependencies"
 run = "npm install"
+sources = ["package.json"]
+outputs = ["node_modules", "package-lock.json"]
 
 [tasks.css]
 description = "generate css"
 depends = ["deps"]
 run = "npx tailwindcss -o ./priv/css/tailwind.css && gzip -k -f ./priv/css/tailwind.css"
-
-[tasks.new]
-description = "generate new project [NAME] from template"
-depends = ["css"]
-run = '''
-gleam new --template=surtoget $1
-cd $1
-gleam run
-'''
+sources = ["src"]
+outputs = ["priv/css"]
 
 [tasks.format]
 description = "gleam format"
-depends = ["css"]
+depends = ["deps"]
 run = "gleam format"
+sources = ["src"]
```

Comment:
As I mentioned in my previous comment, I think we can remove `sources` and `outputs` for this task. The way I see mise works is that it just compares between the last time sources and outputs was edited.

Comparing the last time a file was edited to itself is kinda pointless, so we might as well remove. Otherwise it looks good :)